internal/database/postgres/postgres.go

```diff
@@ -60,8 +60,13 @@ func (psql *Postgres) Features() sqlcommon.SQLFeatures {
 	return features
 }
 
-func (psql *Postgres) UpdateInsertForSequenceReturn(insert sq.InsertBuilder) (sq.InsertBuilder, bool) {
-	return insert.Suffix(" RETURNING seq"), true
+func (psql *Postgres) UpdateInsertForSequenceReturn(insert sq.InsertBuilder, requestConflictEmptyResult bool) (sq.InsertBuilder, bool) {
```

Comment:
This method is confusing in terms of the name ("UpdateInsert") and all the various actions it performs... but it's not a huge deal since it's a pretty internal detail. Just noting it, but no specific actions right now...

Reply:
Went for `ApplyInsertQueryCustomizations` - not sure it's much better